template.yaml

```diff
@@ -206,6 +270,30 @@ Resources:
             Path: /by-owner
             Method: get
             RestApiId: !Ref NvaPublicationApi
+  NvaPublishPublicationFunction:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

You must specify a value that is greater than or equal to 128, and it must be a multiple of 64. You cannot specify a size larger than 3008. Error at Resources/NvaPublishPublicationFunction/Properties/MemorySize